hypershift-operator/controllers/hostedcluster/internal/platform/kubevirt/kubevirt.go

```diff
@@ -0,0 +1,169 @@
+package kubevirt
+
+import (
+	"context"
+
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	"github.com/openshift/hypershift/support/upsert"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	k8sutilspointer "k8s.io/utils/pointer"
+	capikubevirt "sigs.k8s.io/cluster-api-provider-kubevirt/api/v1alpha1"
+	capiv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	hostedClusterAnnotation = "hypershift.openshift.io/cluster"
+	imageCAPK               = "registry.ci.openshift.org/hypershift/cluster-api-kubevirt-controller:1.0.0"
```

Comment:
@nunnatsa :1st_place_medal: 

@nunnatsa can we change the tag? I think something like `0.0.1` or `0.0.1-prerelease` would probably be more accurate since there is not even an alpha release of capik upstream.

Reply:
sure. on it.

Reply:
> sure. on it.

I wonder if we can somehow delete the 1.0.0 tag from the openshift registry. It can potentially be a source of confusion for a long time :)